awx/main/tasks.py

```diff
@@ -1088,6 +1088,7 @@ def run(self, pk, **kwargs):
         '''
         self.safe_env = {}
         private_data_dir = None
+        manager_instance = None
```

Comment:
Can we call this `isolated_manager_instance` everywhere?

Reply:
yep